cfme/tests/configure/test_default_views_infra.py

```diff
@@ -41,6 +42,25 @@ def set_and_test_default_view(group_name, view, page):
     assert tb.is_active(view), "{} view setting failed".format(view)
     DefaultView.set_default_view(group_name, old_default)
 
+
+def check_vm_visibility():
+    view = navigate_to(Vm, 'All')
+    value = view.sidebar.vmstemplates.tree.read_contents()
+    # Selecting the path and last Vm in the accordion to perform click
+    tree_root = value[0]
+    tree_provider = value[1][0][0]
+    tree_datacenter = value[1][0][1][0][0]
+    length = len(value[1][0][1][0][1])
+    tree_vm = value[1][0][1][0][1][length - 1]
+    view.sidebar.vmstemplates.tree.click_path(tree_root,
```

Comment:
If visibility is disabled, this click_path call will raise an exception because `tree_vm` will not be found, will it not?